index.js

```diff
@@ -34,19 +40,43 @@ async function main() {
   const file = flags.get(FILE_FLAG);
   const dir = flags.get(DIR_FLAG);
   const outputDir = flags.get(OUTPUT_DIR_FLAG);
+  const targetBuild = flags.get(TARGET_BUILD_FLAG);
+  const verify = flags.get(VERIFY_FLAG);
 
   if ((file && dir) || (!file && !dir)) {
     throw new Error('Specify exactly one of --file or --dir');
   }
 
   if (file) {
-    return transformFile(file, outputDir);
+    if (transformFile(file, outputDir) === transformResult.SUCCESS && verify) {
+      verifyTransformation(file, targetBuild);
+    }
+    return;
   }
 
   const fileNames = fs.readdirSync(dir);
```

Comment:
nit: put a comment here saying that the code below is for processing a dir

Alternatively (although low priority, so maybe a TODO is fine), it might be nicer if this code always operated on a list of files, and a helper function was responsible for either finding which files to process inside a dir (and returning a list of them), or just returning a single-element list containing `file`